modules/assim.sequential/R/sda.enkf.R

```diff
@@ -307,19 +337,19 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL){
           Xbar = laply(FORECAST[t1:t],function(x){return(mean(x[,i],na.rm=TRUE))})
           Xci  = laply(FORECAST[t1:t],function(x){return(quantile(x[,i],c(0.025,0.975)))})
 
-          plot(total.time[t1:t],Xbar,ylim=range(c(Ybar,Xci),na.rm=TRUE),
+          plot(strftime(obs.times[t1:t],'%Y'),Xbar,ylim=range(c(Ybar,Xci),na.rm=TRUE),
```

Comment:
With sub-annual data this change will cause a mess. If you make sure you pass obs.times through as.Date, R should be able to plot dates on the x-axis just fine.